modules/javafx.graphics/src/main/java/com/sun/glass/ui/win/WinSystemClipboard.java

```diff
@@ -252,7 +252,15 @@ protected final Object popFromSystem(String mimeFull) {
             if (TEXT_TYPE.equals(mime) || URI_TYPE.equals(mime)) {
                 try {
                     // JDK-8118474 - internal Windows data null terminated
-                    return new String(data, 0, data.length - 2, defaultCharset);
+                    // JDK-8281384 - buffer might be larger than data and null terminator not at the end
```

Comment:
Minor: We generally don't include the bug ID of the bug we are fixing in a comment (and I see no need here).

Reply:
Although, since the previous line does, I don't mind if you leave it.